analyzers/tests/SonarAnalyzer.UnitTest/Rules/MethodOverloadsShouldBeGroupedTest.cs

```diff
@@ -28,26 +28,28 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class MethodOverloadsShouldBeGroupedTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<CS.MethodOverloadsShouldBeGrouped>();
```

Comment:
Because there's a VB rule.
```suggestion
        private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.MethodOverloadsShouldBeGrouped>();
```
This is also fine for having the code symetric, even when it's used only once.
```suggestion
        private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.MethodOverloadsShouldBeGrouped>();
        private readonly VerifierBuilder builderVB = new VerifierBuilder<VB.MethodOverloadsShouldBeGrouped>();
```